uint8 axis
int32 input_current # 0.01 Factor
---
bool success